pytorch_lightning/trainer/training_loop.py

```diff
@@ -125,46 +125,15 @@ def setup_fit(self, model, train_dataloader, val_dataloaders, datamodule):
         # check that model is configured correctly
         self.trainer.config_validator.verify_loop_configurations(model)
 
-    def setup_training(self, model: LightningModule):
-        """Sanity check a few things before starting actual training.
-
-        Args:
-            model: The model to run sanity test on.
+    def setup_training(self):
         """
-        # --------------------------
-        # Setup??
-        # --------------------------
+        Sanity check a few things before starting actual training.
+        """
+        model = self.trainer.model
```

Comment:
Let's use model or `model_ref` directly. Not need to do `ref_model = model`

Reply:
I don't think that would be backward compatible, since model on cpu_accelerator has no ref_model, and we need ref_model for summarization and for calling hooks.

Reply:
done.